src/error.rs

```diff
@@ -1,12 +1,10 @@
-#![allow(overflowing_literals)]
-
 use crate::runtime::*;
 use crate::*;
 
 /// The ErrorCode (a.k.a HRESULT) of an error
 #[repr(transparent)]
 #[derive(Copy, Clone, Debug, PartialEq)]
-pub struct ErrorCode(pub i32);
+pub struct ErrorCode(pub u32);
```

Comment:
Is it useful to have a From/Into conversion for Error and i32?

Reply:
I don't expect that to be common with WinRT APIs.